extension/commercial_data/README.md

```diff
@@ -0,0 +1,416 @@
+      items:
+        $ref: 'https://raw.githubusercontent.com/Open-EO/openeo-api/1.0.0/openapi.yaml#/components/schemas/order_id'
+    parameters:
+      type: object
+      description: Key-value pairs of available `order_parameters` as listed at `GET /commercial_data/collections/{collection_id}` for filtering available products.
+      additionalProperties:
+        x-additionalPropertiesName: Order Parameter Name
+        description: Value of the order parameter to be used in the request.
+```
+
+#### `GET /orders/{order-id}`
+
+Get full metadata of the order. The item should follow the [STAC Order Extension](https://github.com/stac-extensions/order), but extended with the spatial and temporal extent information and other metadata about the products.
+
+```json
+{
+  "order:id": "40264b5-c3ae-46f4-a907-0f612d763d97",
+  "order:status": "delivered",
+  "order:date": "2017-01-01T09:32:12Z",
+  "products": [
+    {
+      "type": "Feature",
+      "stac_version": "0.9.0",
+      "id": "c8a1f88d-89cf-4933-9118-45e9c1a5df20",
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [
+              12.36555287044679,
+              41.94403289260048
+            ]
+          ],
+          [
+            12.36571746774068,
+            41.86399361096971
+          ],
+          [
+            12.60746759743069,
+            41.86372776276345
+          ],
+          [
+            12.60758647471871,
+            41.94379931812686
+          ],
+          [
+            12.36555287044679,
+            41.94403289260048
+          ]
+        ]
+      },
+      "properties": {
+        "constellation": "PHR",
+        "acquisitionDate": "2022-03-21T10:11:15.055Z",
+        "azimuthAngle": 179.9852862071639,
+        "cloudCover": 0,
+        "geometryCentroid": {
+          "lat": 41.903935647240964,
+          "lon": 12.486569672582828
+        },
+        "processingLevel": "SENSOR",
+        "sensorType": "OPTICAL",
+        "spectralRange": "VISIBLE"
+      },
+      "assets": {},
+      "links": []
+    }
+  ],
+  "parameters": {
+    "bounds": [
+      3,
+      15,
+      4,
+      16
+    ]
+  },
+  "collection_id": "PLEAIDES",
+  "costs": 42
+}
+```
+
+#### `POST /orders/{order-id}`
+
+When an order is created, the data isn't yet ordered from the commercial data provider. The client should explicitly confirm the order whereupon the order is executed, the costs are deducted from the client's account and the data is ingested in the associated collection. The `order:status` changes to `ordered`.
+
+If the user doesn't have sufficient funds, the endpoint should return an error and `order:status` should not change.
+
+### Payment
+
+Payment should be done in the currency used by the backend, listed at `GET /` under `billing`. When an order is created, the backend should return the full final cost of the order. 
+
+### Example usage with Python client
```

Comment:
I haven't closely checked the Python example yet, I think @soxofaan or another Python client dev should also have a look.